### PR TITLE
Add changes for feature block.

### PR DESCRIPTION
**Pull request by rajathbwsl**

### CODE REVIEW

File: includes/class-responsive-block-editor-addons.php [hunk 1]
```diff
@@ -1159,4 +1167,130 @@ public function cf7_shortcode() {
...
+		$relative_path = 'data/';
+		$full_path = $plugin_dir_path . $relative_path;
+		$file_path_all = $full_path . 'responsive-sites-gutenberg-all.json';
+
+		// Write the JSON data to the file
+		file_put_contents($file_path_all, $filtered_json_all);
+
+		// Check if the data was successfully written to the file
+		if ($file_path_all !== false) {
+			error_log('Filtered data for all pages has been written to: ' . $file_path_all);
```
Comment (pallavic27) [inline includes/class-responsive-block-editor-addons.php]: remove error log

File: readme.txt [hunk 2]
```diff
@@ -199,6 +199,9 @@ This is how you can easily install the Responsive Gutenberg Blocks plugin on you
 
 == Changelog ==
 
+= 1.7.8 - 1st January 2024 =
```
Comment (pallavic27) [inline readme.txt]: check roadmap and update changelog

File: includes/class-responsive-block-editor-addons.php (hunk 1 above)
Comment (rajathbwsl) [reply includes/class-responsive-block-editor-addons.php]: removed it in a new commit.

File: src/blocks/section-block/layout/layout-modal.js
Comment (pallavic27) [inline src/blocks/section-block/layout/layout-modal.js:1]: can we remove commented code

File: readme.txt (hunk 2 above)
Comment (rajathbwsl) [reply readme.txt]: changed the release date to 4th jan.

File: src/blocks/section-block/layout/layout-modal.js
Comment (rajathbwsl) [reply src/blocks/section-block/layout/layout-modal.js:1]: removed commented code.

**Pull request merged**